sphinx_needs/lsp/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class NeedlsConfigException(BaseException):
```

Comment:
this should use PascalCase. also, pep8 says errors should have the suffix 'error' rather than 'exception'.

```suggestion
class NeedLsConfigError(BaseException):
```